vnc.html

```diff
@@ -65,6 +65,12 @@
 </head>
 
 <body>
+
+    <div id="fallback_error">
+        noVNC encountered an error:<br>
```

Comment:
Probably cleaner to give this it's own div instead. It's weird mixing inline and block elements.